patchwork/models.py

```diff
@@ -33,6 +33,14 @@ def validate_regex_compiles(regex_string):
         raise ValidationError('Invalid regular expression entered!')
 
 
+class TimestampMixin(models.Model):
+    created_at = models.DateTimeField(default=tz_utils.now)
+    updated_at = models.DateTimeField(default=tz_utils.now)
```

Comment:
I think you want `auto_now_add` and `auto_now` for `created_at` and `updated_at`, respectively. You should be able to drop `update_timestamp` and the call to same then.

https://docs.djangoproject.com/en/dev/ref/models/fields/#django.db.models.DateField

Reply:
Nice catch. Changed to use these options